src/array.jl

```diff
@@ -615,7 +615,7 @@ similar(::Type{T}, dims::Dims) where {U, T<:Array{Union{CategoricalValue{U}, Mis
     compress(A::CategoricalArray)
 
 Return a copy of categorical array `A` using the smallest reference type able to hold the
-number of [`levels`](@ref) of `A`.
+number of [`levels`](@ref DataAPI.levels) of `A`.
```

Comment:
Does the link `DataAPI.levels` work?

Reply:
Yes it does! :-)

Reply:
Wow - it has not some time ago! This is great.

Reply:
I have just checked the docs. It prints "Missings.levels" as a function name. But this is minor.